solarwinds_apm/apm_constants.py

```diff
@@ -17,7 +17,6 @@
 INTL_SWO_TRACESTATE_KEY = "sw"
 INTL_SWO_X_OPTIONS_KEY = "sw_xtraceoptions"
 INTL_SWO_X_OPTIONS_RESPONSE_KEY = "xtrace_options_response"
-INTL_SWO_SIGNATURE_KEY = "sw_signature"
```

Comment:
All xto info is in the `XTraceOptions` struct under `INTL_SWO_X_OPTIONS_KEY` key in Otel context, so we no longer store under a separate `INTL_SWO_SIGNATURE_KEY`.